### PR TITLE
Update mla8 and mla9

### PR DESCRIPTION
**Pull request by POBrien333**
closes https://github.com/citation-style-language/styles/issues/5352

Let's see if you like this :)

### CODE REVIEW

**Comment by bwiernik** [comment]
What does MLA say about newspapers and magazines for publisher?

**Comment by adam3smith** [comment]
At least in the 8th edition, all periodicas are treated the same: no publisher, no place of publication (for local Newspapers, it can be added in square brackets after the name of the paper, but that should likely be in the data, not automated)

File: modern-language-association-8th-edition.csl [hunk 1]
```diff
@@ -193,7 +193,11 @@
     </group>
   </macro>
   <macro name="publisher">
-    <text variable="publisher"/>
+    <choose>
+      <if type="article-journal" match="none">
```
Comment (bwiernik) [inline modern-language-association-8th-edition.csl:197]:
```suggestion
      <if type="article-magazine article-newspaper article-journal" match="none">
```

File: modern-language-association.csl [hunk 2]
```diff
@@ -194,7 +194,11 @@
     </group>
   </macro>
   <macro name="publisher">
-    <text variable="publisher"/>
+    <choose>
+      <if type="article-journal" match="none">
```
Comment (bwiernik) [inline modern-language-association.csl:198]:
```suggestion
      <if type="article-magazine article-newspaper article-journal" match="none">
```

**Review by bwiernik — CHANGES_REQUESTED** [review]

**Comment by POBrien333** [comment]
Thanks :)

**Comment by adam3smith** [comment]
Thanks! We should think about how to implement this more widely - with Zotero importing publishers via embedded metadata, e.g., we'll see more of this for less prominent styles

**Pull request merged**